examples/example_2.mff/history.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
I've got some concerns about including this file into the master directory - as it directly references a user, a location, and a subject identifier.

To be on the safe side regarding PIIA, can we try to scrub those strings from the XML? See if the data still works.